pkg/reaper/nodereaper/nodereaper.go

```diff
@@ -657,6 +655,9 @@ func (ctx *ReaperContext) reapOldNodes(w ReaperAwsAuth) error {
 		if err != nil {
 			return err
 		}
+
+		log.Infof("starting deletion throttle wait -> %vs", ctx.AgeReapThrottle)
```

Comment:
Do you want to do this when it's a DryRun?

Reply:
Good call! Will do